### PR TITLE
Update .travis.yml

### PR DESCRIPTION
**Pull request by mforets**

### CODE REVIEW

**Comment by schillic** [comment]
There was an [error](https://travis-ci.com/github/JuliaReach/ReachabilityAnalysis.jl/jobs/309465490#L652) with the coverage.
```
ERROR: Base.Meta.ParseError("extra token \"\"\" after end of expression")
Stacktrace:
 [1] #parse#1(::Bool, ::Bool, ::Bool, ::typeof(Base.Meta.parse), ::String, ::Int64) at ./meta.jl:184
 [2] parse(::String, ::Int64) at ./meta.jl:176
 [3] amend_coverage_from_src!(::CoverageTools.FileCoverage) at /Users/travis/.julia/packages/CoverageTools/mRqnP/src/CoverageTools.jl:189
 [4] process_file(::String, ::String) at /Users/travis/.julia/packages/CoverageTools/mRqnP/src/CoverageTools.jl:240
 [5] process_folder(::String) at /Users/travis/.julia/packages/CoverageTools/mRqnP/src/CoverageTools.jl:263
 [6] process_folder(::String) at /Users/travis/.julia/packages/CoverageTools/mRqnP/src/CoverageTools.jl:269 (repeats 2 times)
 [7] process_folder() at /Users/travis/.julia/packages/CoverageTools/mRqnP/src/CoverageTools.jl:255
 [8] top-level scope at none:1
```

File: .travis.yml [hunk 1]
```diff
@@ -25,3 +25,4 @@ notifications:
 after_success:
   - julia --color=yes --project=test/coverage/ -e 'using Pkg; Pkg.instantiate()'
   - julia --color=yes --project=test/coverage/ test/coverage/coverage.jl
+  - julia -e 'using Pkg; Pkg.add("Coverage"); using Coverage; Codecov.submit(process_folder())'
```
Comment (schillic) [inline .travis.yml]: Are both lines needed?

**Comment by mforets** [comment]
> There was an error with the coverage.

thanks for checking. the last commit should have fixed that error. the stacktrace doesn't say too much, but manually running `process_folder("src/Algorithms/LGG09")` i found the problem.

**Comment by mforets** [comment]
that did fix the coverage. thanks!

**Pull request merged**